cpp/include/cudf/strings/find_multiple.hpp

```diff
@@ -28,8 +28,42 @@ namespace strings {
  */
 
 /**
- * @brief Returns a lists column with character position values where each
- * of the target strings are found in each string.
+ * @brief Searches for the given target strings within each string in the provided column
+ *
+ * Each column in the result table corresponds to the result for the target string at the same
+ * ordinal. i.e. 0th column is the BOOL8 column result for the 0th target string, 1th for 1th,
```

Comment:
```suggestion
 * ordinal. i.e. 0th column is the BOOL8 column result for the 0th target string, 1st for 1st,
```

Reply:
At the risk of sounding like I'm quibbling, wouldn't `1st` (first) indicate the `0th` ordinal?  By extension, `1th` should be the `2nd` (second) one.  In my eyes, `1st` and `1th` are off by, well, `1`.  

That said, the comment in this final version is legible, so let's leave it here.

Reply:
Thanks. Given the description includes mention of the `0th` entries, I'm not worried about the `1st for 1st` being confusing in this context.